lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -538,6 +538,39 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 	*/
 	setReadOnly( TypedArray.prototype, 'BYTES_PER_ELEMENT', TypedArray.BYTES_PER_ELEMENT );
 
+	/**
+	* Invokes a function once for each array element.
+	*
+	* @name findIndex
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @param {Function} callbackFn - function to invoke
```

Comment:
```suggestion
	* @param {Function} predicate - predicate function
```

I suggest renaming here and below.